### PR TITLE
DBaaS UI fixes

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

- Standardizes the capitalization for "Database Cluster"
- Fixes the double border on the last row of IP addresses for Access Controls
- Removes the line break from the Delete Cluster copy
- Add "Root" to the primary action for the Reset Root Password modal

## How to test

- Check all the copy references to "Database Cluster" and check for capitalization
- Check the other changes on the Settings tab

### CODE REVIEW

File: packages/manager/src/features/Databases/DatabaseDetail/AccessControls.tsx [hunk 1]
```diff
@@ -53,13 +53,15 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-    '&:last-child': {
-      borderBottom: 'none',
-    },
     [theme.breakpoints.down('xs')]: {
       width: '100%',
     },
   },
+  row: {
+    '&:last-of-type td': {
+      borderBottom: 'none',
```
Comment (DevDW) [inline packages/manager/src/features/Databases/DatabaseDetail/AccessControls.tsx:62]: 🔥🔥🔥

File: packages/manager/src/features/Databases/DatabaseCreate/DatabaseCreate.tsx [hunk 2]
```diff
@@ -529,7 +532,7 @@ const DatabaseCreate: React.FC<{}> = () => {
           <Grid item xs={12} md={8}>
             <Notice success className={classes.notice}>
               <strong>
-                Notice: There is no charge for database clusters during beta.
+                Notice: There is no charge for Database Clusters during beta.
               </strong>{' '}
               You will be notified before the beta period ends and database
               clusters are subject to charges.{' '}
```
Comment (DevDW) [inline packages/manager/src/features/Databases/DatabaseCreate/DatabaseCreate.tsx:538]: have to capitalize here as well

**Review by DevDW — APPROVED** [review]
One instance of "database clusters" needs capitalization still but everything looks good aside from that 👍

**Review by HanaXu — COMMENTED** [review]
Looks like there's some merge conflicts that need to be addressed

Also can we fix this as well? 
image

**Review by HanaXu — APPROVED** [review]

**Review by bnussman — APPROVED** [review]
One last place that could use an update would be this. I think `database` should be `Database Cluster`...
Screen Shot 2022-02-03 at 5 42 36 PM

Other than that potential copy update, this looks good!

**Pull request merged**